src/bin/webpack-validator.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+'use strict'
+const fs = require('fs')
+const path = require('path')
+const program = require('commander')
+const log = require('npmlog')
+const validate = require('../')
+const schema = require('../').schema
+let configFile
+
+program
+  .arguments('<configFileName>')

```

Comment:
In my very first version I defaulted to webpack.config. But as I plan to support power users via .webpack-validatorrc I decided to postpone defaults (and power user usage)t o the next version. This is a basic version and of course it will get improved via feedback but fur now I think it already provides value